ansible/roles/etcd/tasks/etcd2-start.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: CoreOS | Enable etcd2
+  service: name=etcd2 enabled=yes
```

Comment:
Make this conditional? Introduce ``enable_services: true|false`` in ``group_vars/all.yml``? When I run the ansible in CI environment, the cluster is destroyed right after it is used. Thus, no need to enable/disable any service. This will not save much time but it could decrease dependency on systemd.